server/src/main/resources/spring/login-ui.xml

```diff
@@ -554,6 +554,7 @@
     <bean id="changePasswordService" class="org.cloudfoundry.identity.uaa.account.UaaChangePasswordService">
         <constructor-arg ref="scimUserProvisioning"/>
         <constructor-arg ref="uaaPasswordValidator"/>
+        <constructor-arg value="${password.historyRestriction:1}"/>
```

Comment:
A history of 1 mimics existing UAA behavior in which you cannot update the password to the same value.